docs/DeveloperGuide.md

```diff
@@ -158,21 +158,51 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Lesson Time Parameter
+This parameter allows users to keep track of a student's lesson timings. Multiple lesson times can be added for a single student.
+
+#### Design Considerations
+**Treatment of clashing timings**
+
+- **Current Implementation (Alternative 1):**
+    - **Description**: Clashing lesson timings across and within students are not rejected.
+    - **Pros**: Promotes greater user flexibility, as there could be different schedules for alternate weeks,
```

Comment:
Provides greater user flexibility, accommodating different schedules for alternate weeks or varying lesson arrangements and allows all specified timings to be stored in EduManage, even if they overlap.